orchestra/management/commands/runstaffbot.py

```diff
@@ -0,0 +1,29 @@
+import time
+
+from django.core.management.base import BaseCommand
+from django.conf import settings
+from django.utils import autoreload
+from slackclient import SlackClient
+
+from orchestra.bots.staff_bot import StaffBot
+from orchestra.core.errors import SlackConnectionError
+
+
+class Command(BaseCommand):
+    help = 'Runs a Slack StaffBot for RTM calls'
+
+    def handle(self, *args, **options):
+        autoreload.main(self.inner_run, None, options)
+
+    def inner_run(self, *args, **options):
+        slack_client = SlackClient(settings.STAFF_BOT_API_KEY)
+
+        if slack_client.rtm_connect():
+            staff_bot = StaffBot(slack_client)
+            while True:
```

Comment:
how would we use this in production? what port is it listening on? do we have to spin up another server?